importpatches.py

```diff
@@ -1,3 +1,5 @@
+#! /usr/bin/env python
```

Comment:
This script is expected to run outside of virtual environment. Let's be specific and follow our own advice here?

```suggestion
#! /usr/bin/env python3
```